doc/source/getting_started/learning.rst

```diff
@@ -150,6 +176,15 @@ showcases, visit `Technology showcase guide <ansys_techdemo_guide_>`_.
 
 **NOTE**: You might need to create a free Ansys account to access these resources.
 
+
+Additional resources
+====================
+
+Ansys has prepared multiple resources to help you to learn and use PyMAPDL.
+You can access them from the website `Ansys Learning Resources <learning_resources_>`_.
+In this page, you can find a summary of the available resources for MAPDL and PyMAPDL.
```

Comment:
On this page--but since the previous sentence is using website, it seems that this would be better: "On this website, you ...."